### PR TITLE
XW-4241 | remove wikia-style-guide from deps

### PR DESCRIPTION
**Pull request by hakubo**
CSS size: 35.3kb -> 24.3kb compressed gzipped

### CODE REVIEW

File: app/styles/component/_article-featured-video.scss [hunk 1]
```diff
@@ -79,11 +79,3 @@
 		}
 	}
 }
-
-.jwplayer {
-	// resets wikia-style-guide styles
-	input[type="checkbox"] + label {
```
Comment (latata) [inline app/styles/component/_article-featured-video.scss:85]: 🥇

**Review by latata — APPROVED** [review]

File: app/mixins/render-component.js [hunk 2]
```diff
@@ -5,7 +5,7 @@ export default Mixin.create({
 	didInsertElement() {
 		if (this._placeholderElement) {
 			this._placeholderElement.parentNode.insertBefore(this.element, this._placeholderElement);
-			$(this._placeholderElement).remove();
+			this._placeholderElement.remove();
```
Comment (bkoval) [inline app/mixins/render-component.js]: Seems like node.remove() might not be supported by IE11 (https://developer.mozilla.org/pl/docs/Web/API/ChildNode/remove) - how about this._placeholderElement.parentElement.removeChild(this._placeholderElement)?

File: app/models/widget-discussions.js [hunk 3]
```diff
@@ -52,7 +52,11 @@ export default EmberObject.extend(
 						name: createdBy.name,
 						profileUrl: buildUrl({
 							namespace: 'User',
-							title: createdBy.name
+							title: createdBy.name,
+							relative: true,
+							query: {
+								useskin: 'oasis'
```
Comment (bkoval) [inline app/models/widget-discussions.js:58]: Why do we need it on mobile?

File: app/styles/_wikia-variables.scss [hunk 4]
```diff
@@ -42,7 +42,7 @@ $z-highlight-overlay-content: $z-9;
 //////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
 
 // Border Radius
-$border-radius-global: rem-calc(2);
+$border-radius-global: .125rem;
```
Comment (bkoval) [inline app/styles/_wikia-variables.scss:45]: :+1: for straight usage of rems

File: app/styles/base/_body.scss [hunk 5]
```diff
@@ -3,6 +3,17 @@ html {
 }
 
 body {
+	background: #fff;
```
Comment (bkoval) [inline app/styles/base/_body.scss]: We did have a $body-background variable (removed in this PR). I would consider either bringing it back or using $white here and then a corresponding _wikia-variables color var for text color.

File: app/styles/base/_body.scss [hunk 6]
```diff
@@ -3,6 +3,17 @@ html {
...
+	color: $body-font-color;
+	cursor: auto;
+	font-style: normal;
+	line-height: 150%;
 	overflow-y: scroll;
+	position: relative;
 	word-wrap: break-word;
 }
+
+* {
```
Comment (bkoval) [inline app/styles/base/_body.scss]: Shouldn't it be covered by reset?

File: app/styles/base/_typography.scss [hunk 7]
```diff
@@ -0,0 +1,73 @@
+blockquote {
+	border-left: .25rem solid rgba(26, 94, 184, .3);
+	color: #3a3a3a;
```
Comment (bkoval) [inline app/styles/base/_typography.scss:3]: Maybe adding a wikia-variable for it would make this clearer?

File: app/styles/base/_typography.scss [hunk 8]
```diff
@@ -0,0 +1,73 @@
...
+	font-size: 1rem;
+	font-style: italic;
+	font-weight: 400;
```
Comment (bkoval) [inline app/styles/base/_typography.scss]: How about $font-weight-normal?

File: app/styles/base/_typography.scss [hunk 9]
```diff
@@ -0,0 +1,73 @@
...
+	line-height: 1.625rem;
```
Comment (bkoval) [inline app/styles/base/_typography.scss]: $line-height-base :)

File: app/styles/base/_typography.scss [hunk 10]
```diff
@@ -0,0 +1,73 @@
...
+		padding-top: .875rem;
+
+		&::before {
+			content: "\2014 \0020";
+		}
+	}
+}
+
+a {
+	color: #1a5eb8;
```
Comment (bkoval) [inline app/styles/base/_typography.scss]: $color-blue-light

File: app/styles/component/_widget-discussion.scss [hunk 11]
```diff
@@ -49,6 +49,12 @@
 		.avatar {
 			display: inline-block;
 			margin-right: 5px;
+
+			img {
+				border: 2px solid #fff;
```
Comment (bkoval) [inline app/styles/component/_widget-discussion.scss]: $white

**Review by bkoval — CHANGES_REQUESTED** [review]

File: app/mixins/render-component.js (hunk 2 above)
Comment (hakubo) [reply app/mixins/render-component.js]: I don't think we support IE11 on mobile

File: app/models/widget-discussions.js (hunk 3 above)
Comment (hakubo) [reply app/models/widget-discussions.js:58]: to avoid some round trips 
with this we'll go straight to oasis

**Review by bkoval — APPROVED** [review]

**Pull request merged**